lib/gc/ExecutionEngine/GPURuntime/ocl/GpuOclRuntime.cpp

```diff
@@ -808,6 +808,10 @@ OclModuleBuilder::build(const OclRuntime::Ext &ext) {
   opts.enablePerfNotificationListener = false;
 #endif
 
+  if (enableObjectDump) {
+    mod.dump();
+  }
+
```

Comment:
@AndreyPavlenko what do you think, should it be a separate option in `OclModuleBuilderOptions`? Or maybe we can add a bool parameter to `OclModuleBuilder::build(..., bool dump_lowered_module=false)`?

Reply:
It should be a separate option, because enableObjectDump is used to dump the compiled object.
It could also be implemented without adding a new option, like here - https://github.com/intel/graph-compiler/blob/bf27ed508d27274505abec13c9c868c3e962d8ef/src/gc-gpu-runner/GpuRunner.cpp#L159